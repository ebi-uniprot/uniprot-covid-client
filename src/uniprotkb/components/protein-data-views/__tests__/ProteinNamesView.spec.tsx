import React from 'react';
import { render } from '@testing-library/react';
import EntryProteinNames from '../ProteinNamesView';
import ProteinNamesUIData from '../../../__mocks__/proteinNamesUIData.json';

describe('ProteinNames', () => {
  test('should render protein_name', () => {
    const { asFragment } = render(
      <EntryProteinNames {...ProteinNamesUIData} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
