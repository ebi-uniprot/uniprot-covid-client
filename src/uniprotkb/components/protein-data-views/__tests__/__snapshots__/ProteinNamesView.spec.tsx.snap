// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProteinNames should render protein_name 1`] = `<DocumentFragment />`;
